src/services/google.json

```diff
@@ -23,11 +23,11 @@
   "tosback2": {
     "definitions": {
       "name": "Key terms",
-      "url": "https://www.google.com/intl/en/policies/privacy/key-terms/"
```

Comment:
This will probably change back automatically next time you import and export from tosdr2. I think this appeared because we did the import into tosdr2 before the `git pull` of tosdr-build.